runtime/integration-tests/src/integrations_mock/mod.rs

```diff
@@ -48,10 +48,11 @@ cfg_if::cfg_if! {
         type RuntimeAssetConfig = calamari_runtime::assets_config::CalamariAssetConfig;
         type RuntimeConcreteFungibleLedger =
             calamari_runtime::assets_config::CalamariConcreteFungibleLedger;
+        const UNIT: Balance = KMA;
     } else {
         pub mod test_manta;
         use manta_runtime::{
-            currency::MANTA as KMA,
+            currency::MANTA,
```

Comment:
why not just `use currency::MANTA as UNIT` ?

Reply:
Well the thought is that then `KMA` can be used in calamari specific tests and `MANTA` can be used in manta tests and then `UNIT` is used for common tests.

But maybe just changing all tests to use `UNIT` is cleaner. I am cool w either way

Reply:
I'd personally prefer `UNIT` across all tests, but that's just my preference. Ok leaving as it is